torch/csrc/generic/StorageMethods.cpp

```diff
@@ -161,6 +161,8 @@ static PyObject * THPStorage_(fromBuffer)(PyObject *_unused, PyObject *args, PyO
 #if defined(TH_REAL_IS_BYTE) || defined(TH_REAL_IS_CHAR)
   memcpy(THWStorage_(data)(storage), src + offset, count);
 #elif defined(TH_REAL_IS_BOOL)
+  // Because of ASAN checks, we have to manually move bytes instead of
```

Comment:
you aren't moving bytes though, you are changing the the values.  How about include the ASAN check that tripped this?